src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1,4 +1,83 @@
 databaseChangeLog:
   -  changeSet:
        id:  adicionar suporte ao liquibase
-       author:  izumizawa
\ No newline at end of file
+       author:  izumizawa
+
+  -  changeSet:
+       id:  Adding telephone type table
+       author:  izumizawa
+       changes:
+         -  createTable:
+              tableName: telephone_type
+              columns:
+                - column:
+                    name: telephone_id_type
+                    type: int
+                    constraints:
```

Comment:
Can you please add a PK name here? like "telefone_type_pk".
You can also remove the nullable constraint as it's is redundant, PKs can't be null by default :)